app/views/rooms/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>Room Index</h1>
+<% @rooms.each do |room|%>
+  <div id="room-<%= room.id %>">
+    Room#: <%= room.id %> --- 
+    Suite: <%= room.suite%> ---
+    Hotel: <%= room.hotel.name %> ---
+    Room Guests: <%= room.guests.count %>
```

Comment:
We'd rather you encapsulate the guest counting to a model method in the Room model for a couple reasons
1) As a general rule, it's a calculation (although minor) so it should remain close to the data rather than living in presentation logic
2) The counting could be more complex. For example, you could include `distinct` or other conditions. Encapsulating it in the model allows you to test it easily in a more isolated environment. 

The rubric allows for one MVC infraction so this doesn't affect your score in that category